src/backend/renderer/mod.rs

```diff
@@ -227,6 +234,11 @@ pub trait Renderer {
     /// Set the filter method to be used when rendering a texture into a larger area than its size
     fn upscale_filter(&mut self, filter: TextureFilter) -> Result<(), Self::Error>;
 
+    /// Set the enabled [`DebugFlags`]
+    fn set_debug_flags(&mut self, flags: DebugFlags);
```

Comment:
Could it be useful to check which debug flags are supported?

Reply:
Something like returning the flags from something like `supported_debug_flags() -> DebugFlags`?